src/OrbitBase/Tracing.cpp

```diff
@@ -79,16 +79,14 @@ static std::vector<TracingScope>& GetThreadLocalScopes() {
   return thread_local_scopes;
 }
 
-namespace orbit_api {
-
-void Start(const char* name, orbit::Color color) {
+void orbit_api_start(const char* name, orbit_api_color color) {
```

Comment:
Is there a special reason for the snake_case syntax? Feels a bit inconsistent. At least I would expect a comment.

Reply:
The reason is to signal that those are "C" functions, not "C++" anymore.